src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1152,12 +1189,16 @@ private void CheckboxPrerelease_CheckChanged(object sender, EventArgs e)
             }
 
             var timeSpan = GetTimeSinceLastRefreshAndRestart();
+
             RegistrySettingUtility.SetBooleanSetting(Constants.IncludePrereleaseRegistryName, _topPanel.CheckboxPrerelease.IsChecked == true);
             EmitRefreshEvent(timeSpan, RefreshOperationSource.CheckboxPrereleaseChanged, RefreshOperationStatus.Success);
 
             NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
             {
-                await SearchPackagesAndRefreshUpdateCountAsync(useCacheForUpdates: false);
+                using (_pmuiGestureintervalTracker.Start(nameof(CheckboxPrerelease_CheckChanged)))
```

Comment:
IntervalTracker cannot handle overlapping events (it only has 1 stopwatch)

Is it possible that these events overlap?

Reply:
That means the way you read it as "this event took DURATION_MSEC time itself",  but child events are a part of that, not in addition to it.

Reply:
Moved back to draft... definitely need to come up with a new paradigm here.